setup.py

```diff
@@ -32,5 +33,5 @@
         "Topic :: Utilities",
     ],
     python_requires=">=3.5",
-    install_requires=["Django>=2.2,<3.1", "requests"],
+    install_requires=["Django>=2.2,<3.2", "requests"],
```

Comment:
IMO, the upper bound should be unpinned to allow testing against Django master, see #30

Reply:
I think you're right, but the change should probably go through your PR #30 ?

Reply:
That approach works for me. Thanks.